### ISSUE TITLE
The scores are only -1.

### ISSUE THREAD

**Issue by Joel-MalwareBenchmark** [fizyr/keras-retinanet#698]
In this case, I attempt to used csv file to train the model. However, even  when I used the trained data to test, it could not detect. 
 loss: 4.1363 - regression_loss: 3.0210 - classification_loss: 1.1154
I used 50 pictures as one class to trained, 10epochs

![000005](https://user-images.githubusercontent.com/33394474/46067689-0a86af00-c1aa-11e8-9c70-c8de8eea832f.jpg)

**Comment by PhilippMarquardt** [comment]
The loss is quite high. Do you have enough training data? What objects are you trying to detect? If you have 50 images it doesn't matter if there is only one object of interest on each image.

**Comment by fernandocamargoti** [comment]
Have a look here and see if you're having the same problem:

https://github.com/fizyr/keras-retinanet/issues/694

**Comment by hgaiser** [comment]
I'm closing this as it is not an issue with the codebase. Please use the Slack channel for usage information.
